test/integration/elasticquota_controller_test.go

```diff
@@ -37,9 +37,9 @@ import (
 	fwkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
 	st "k8s.io/kubernetes/pkg/scheduler/testing"
 
-	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling"
-	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
-	schedv1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+	"sigs.k8s.io/scheduler-plugins/apis/scheduling"
```

Comment:
looks OK. I wonder: how costly is to cherry-pick the commit(s) that fixed this u/s?

Reply:
We could certainly do that. Here is the commit https://github.com/kubernetes-sigs/scheduler-plugins/pull/340/commits/b5ea0f1f188e37f770868c7a74a9ae91107dd4e1. Would you prefer it to be done as part of this PR?

Reply:
> We could certainly do that. Here is the commit [kubernetes-sigs@b5ea0f1](https://github.com/kubernetes-sigs/scheduler-plugins/commit/b5ea0f1f188e37f770868c7a74a9ae91107dd4e1). Would you prefer it to be done as part of this PR?

if cherry-picking this commit happens cleanly and if is not too much hassle, I'd prefer to cherry-pick it and keep the history as close as possible. Most of these changes are not compiled in in our artifacts, so it's almost zero extra risk.